addendum.dict

```diff
@@ -24,6 +24,7 @@ Aung	𐑷𐑙
 Aurea	𐑹𐑾
 Autoshop	𐑷𐑑𐑴𐑖𐑪𐑐
 bachelorette	𐑚𐑨𐑗𐑩𐑤𐑼𐑧𐑑
+baguette 𐑚𐑨𐑜𐑧𐑑
```

Comment:
Added to main dictionary kingsleyreadlexicon.tsv, and can be removed from this pull request.